store/storeutil/storeutil.go

```diff
@@ -74,7 +74,7 @@ func GetCurrentInstance(txn *store.Txn, dockerCli command.Cli) (*store.NodeGroup
 func GetNodeGroup(txn *store.Txn, dockerCli command.Cli, name string) (*store.NodeGroup, error) {
 	ng, err := txn.NodeGroupByName(name)
 	if err != nil {
-		if !os.IsNotExist(errors.Cause(err)) {
+		if !os.IsNotExist(errors.Cause(err)) && !store.IsErrInvalidName(err) {
```

Comment:
`errors.Is`?

Reply:
Would need to make the interface public and also error contains a dynamic value.

Reply:
Yeah good point, that sounds alright for now. We're not consistent on styles in buildx anyways, so :shrug:

Reply:
Agree we should improve this in our code base with well defined error types.